zenodo/modules/utils/files.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Actually, It is the part of your commit as `zenodo/researcher_profile` branch is behind master.